scripts/set-git-rev/default.nix

```diff
@@ -34,7 +34,7 @@ let
     '') drvOutOutputs);
 
   setGitRev = pkgs.runCommand "set-git-rev" {} ''
-    ${ghc.withPackages (ps: [ps.universum ps.file-embed])}/bin/ghc -o $out ${./set-git-rev.hs}
```

Comment:
We should reference the upstream issue, which is https://github.com/NixOS/nixpkgs/issues/46814